app/src/main/java/com/ataulm/artcollector/HarvardArtMuseumApi.kt

```diff
@@ -14,11 +14,14 @@ interface HarvardArtMuseumApi {
     @GET("object?$PAINTINGS_&$WITH_IMAGES_&$WITH_ARTIST_&$INC_FIELDS")
     fun gallery(): Deferred<ApiPaintingsResponse>
 
+    @GET("person")
+    fun artist(@Query("q") qValue: String): Deferred<ApiPersonResponse>
```

Comment:
the query value is something like `"personid:${artistId.value}"` which I don't know how to do in Retrofit without passing the whole string to this method.